docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+**The breaking changes made in v5**
+
+Our most recent release, MUI Core v5, introduced some important breaking changes. The main problem we wanted to tackle with it was customizability, something we have been hearing as an improvement opportunity for a while. The changes done are meant to unlock more styling capacity while not deprioritizing performance. We understand that it introduced a considerable amount of work to migrate but, we’re still in the early days of maturing this new styling solution. There are a lot of opportunities to expand on the reason why the change was made, how to get the most out of it, best practices, tutorials, automation tools, and, generally, how-to guide those concerned about migrating to v5 because of it. If you have any ideas or suggestions, you’re always welcome to share your thoughts by opening an issue on our repository.
+
+### How likely is it that you would recommend MUI to a friend or colleague?
+
+As for the Net Promoter Score, this year we also saw a decrease. Last year, we had 62,2% of the respondents as Promoters whereas now we had 56,4%. NPS itself moved from 62 in 2020 to 46 in 2021. If we consider the scale, we're still at the great level, though (30 to 70 is considered great), which makes us glad. We definitely have much more to do to achieve an excellent score. The reasoning for the previous and next question might fit to explain NPS data.
+
+<img src="/static/blog/2021-survey/2.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Detractors: 10.88%, Neutrals: 32.69%, Promoters: 56.43%, Overall NPS: 46. Scale: -100 to 0 needs improvements, 0 to 30 is good, 30 to 70 is great, and 70 to 100 is excellent." />
+
+<p class="blog-description">1563 out of 1589 answered</p>
+
+### What is the main benefit you get from using MUI?
+
+<img src="/static/blog/2021-survey/3.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="This question results in 2019 - Design: 248; Time: 193; DX: 128; Components: 89; Customizability: 53; Documentation: 25; Accessibility: 5; Typescript: 5; Community support: 4; This question results in 2021 - Time: 597; Design: 407; Components: 309; DX: 306; Customizability: 181; Documentation: 57; Community support: 26; Accessibility: 24; Performance: 17" />
+
+<p class="blog-description">1422 out of 1589 answered this question in 2021</p>
+
+Here's what grew in terms of value proposition:
+
+- **Community (x2.9):** On a similar time frame, community size grew by x2.0. It looks like we're witnessing a network effect: with x2 more users, we got x3 more value from the community to the user base.
```

Comment:
Is this a standard style choice to express the growth numbers as x2.9? Maybe this is a US vs. the world kind of thing, but it makes a lot more sense to me when these numbers are expressed as +290% for example. 

I would also expect to see "2x more developers" instead of "x2 more," but in this case I think I would opt for "twice" and "three times" instead.


```suggestion
- **Community (x2.9):** While appreciation for the community grew by 290%, the community itself doubled in size over that same time frame. It looks like we're witnessing a network effect: with twice as many users, we got three times more value from the community.
```

Reply:
Uhm, yeah. I think it's more an English vs Romance languages where the multiplier comes before and not after the number. That could be probably why for me (Portuguese) and Olivier (French) it didn't sound weird. But happy to go with the standard US English approach.